ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -99,17 +107,25 @@ const TopologyComponent = () => {
         };
     }, [controller]);
 
+    const selectedIds = selectedObj ? [selectedObj.id] : [];
+
     return (
         <TopologyView
             sideBar={
-                <TopologySideBar show={!!selectedId} resizable onClose={() => setSelectedId(null)}>
-                    <div style={{ height: '100%' }}>{selectedId}</div>
+                <TopologySideBar
+                    show={!!selectedObj}
+                    resizable
+                    onClose={() => setSelectedObj(null)}
+                >
+                    {selectedObj?.type === 'group' && <div>Group</div>}
+                    {selectedObj?.type === 'node' && <DeploymentSideBar />}
+                    {selectedObj?.type === 'edge' && <div>Edge</div>}
                 </TopologySideBar>
             }
-            sideBarOpen={!!selectedId}
+            sideBarOpen={!!selectedObj}
```

Comment:
you can have the side bar be open but not shown? that seems a little confusing to me. is that case for like a minimized side panel?

Reply:
Do you mean how TopologyView has the sideBarOpen prop and the TopologySideBar has the show prop? I tested it out and seems like we need sideBarOpen but the show prop didn't seem necessary. I removed that one